sql/main/ddl.sql

```diff
@@ -74,6 +75,21 @@ BEGIN
         END IF;
     END IF;
 
+    EXECUTE format('SELECT TRUE FROM _timescaledb_catalog.hypertable WHERE
+                    hypertable.schema_name = %L AND
+                    hypertable.table_name = %L',
+                    schema_name, table_name) INTO is_hypertable;
+
+    IF is_hypertable THEN
```

Comment:
Should this replace the check in line 118 below. I think we should remove that other check below if it's not necessary anymore.